Напишите класс Fraction, конструктор которого принимает числитель и знаменатель. Если пользователь передал в качестве знаменателя 0, то выбрасывайте исключение типа std::runtime_error (которое находится в заголовочном файле stdexcept). В функции main() попросите пользователя ввести два целых числа. Если числа, которые ввел пользователь, корректные, то выводите создаваемый объект класса Fraction. Если же числа недопустимые, то вы должны обрабатывать исключение типа std::exception и сообщить пользователю, что он ввел некорректные данные.

Подсказка: std::runtime_error является дочерним классу-исключению std::exception, поэтому у вас должен быть только один блок catch.

Пример выполнения программы:

Enter the numerator: 7
Enter the denominator: 0
Your fraction has an invalid denominator.

Ответ
